packages/builders/src/node/build/node-build.builder.ts

```diff
@@ -18,6 +18,7 @@ export interface BuildNodeBuilderOptions {
   outputPath: string;
   tsConfig: string;
   watch?: boolean;
+  sourceMap?: boolean;
```

Comment:
If you put the source map to default with `true`, you can remove the weak type.

Reply:
Making it required is technically true for the source code but unfortunately, it makes the tests harder.

We can revisit it, but the tests are written in a way where it expects a full options and doesn't use the smart defaults... I think if we use the architect api within the tests, it would accept `Partial<Options>`.